.gitignore

```diff
@@ -7,3 +7,5 @@ Cargo.lock
 
 # Configurations for VSCode
 .vscode/
+
+/proptest-regressions
```

Comment:
I don't think that's a good idea: we'll eventually want to commit some rng seeds in there.